lrauv_ignition_plugins/proto/lrauv_range_bearing_response.proto

```diff
@@ -33,8 +33,8 @@ message LRAUVRangeBearingResponse
   /// \brief Request ID. Each request ID must be unique.
   uint32 req_id = 1;
 
-  /// \brief Bearing in terms of (r, theta, phi). This is based on ground truth
-  /// and not calculated using time-of-flight
+  /// \brief Bearing in terms of (r, elevation, azimuth). This is based
+  /// on ground truth and not calculated using time-of-flight
```

Comment:
As we discussed, let's replace the bearing vector with range, azimuth, and elevation members with commentary describing the frame of ref and units.

Reply:
Also, let's replace `double range = 3;` in 42 with the actual time-of-flight in seconds

Reply:
These changes could (should?) be worked into a separate PR

Reply:
Yeah I've opened a new PR for these changes see #183

Reply:
Thanks, @arjo129. Then let's fix the debug output in the other comment and should be good to go